bonus/bitcoin/ordisrespector.md

```diff
@@ -186,7 +172,7 @@ gpg: no ultimately trusted keys found
 * Verify that the checksums file is cryptographically signed by the release signing keys. The following command prints signature checks for each of the public keys that signed the checksums
 
 ```sh
-gpg --verify SHA256SUMS.asc
+gpg --verify SHA256SUMS.asc SHA256SUMS.asc
```

Comment:
Why duplicate this? Was it a mistake?

Reply:
Yes. I wanted to change it to `gpg --verify SHA256SUMS.asc SHA256SUMS`. Without the 2nd argument gpg gives this warning: `gpg: assuming signed data in 'SHA256SUMS'`